### DISCUSSION TITLE
problems with member access

### DISCUSSION THREAD

**Discussion by vnetesov** [docmost/docmost#286] [Help / Questions] [answered]
Members do not have access to view page contents in spaces, they only see page headers.
What could be the problem?

<img width="559" alt="изображение" src="https://github.com/user-attachments/assets/612d737c-9059-45fe-b6e6-208062cb0741">

**Comment by Philipinho** [comment]
What version are you on? Do you have websockets enabled?

**Reply by vnetesov** [reply]
> What version are you on? Do you have websockets enabled?

How do I enable a websockets? I didn't see it in the instructions

version 0.3.1 
<img width="774" alt="изображение" src="https://github.com/user-attachments/assets/771546d5-02d2-4538-982e-da296be33d1a">

**Reply by crendrik** [reply, marked as answer]
Hello, 

On the Reverse Proxy Rules click the Custom Header tab. Click Create and then, from the drop-down menu, click WebSocket. After you click on WebSocket, two Header Names and two Values will be automatically added. Click Save. Follow the instructions in the image below.

Step 7 -> https://mariushosting.com/how-to-install-docmost-on-your-synology-nas/

**Reply by vnetesov** [reply]
> Hello,
> 
> On the Reverse Proxy Rules click the Custom Header tab. Click Create and then, from the drop-down menu, click WebSocket. After you click on WebSocket, two Header Names and two Values will be automatically added. Click Save. Follow the instructions in the image below.
> 
> Step 7 -> https://mariushosting.com/how-to-install-docmost-on-your-synology-nas/

Thank you! Is there an instruction for nginx + ubuntu?

**Reply by crendrik** [reply]
Docker in Nas Synology

**Reply by Philipinho** [reply]
@vnetesov for NGINX, see my response here https://github.com/docmost/docmost/discussions/144#discussioncomment-10244716.

**Reply by vnetesov** [reply]
Thanks! The error with the websockets has gone away, but the members still only see the page headers. What else could be wrong?

**Reply by Philipinho** [reply]
Check the “WS” browser console tab if `/collab` websocket path is transmitting data when you type on the editor.

**Reply by vnetesov** [reply]
> Check the “WS” browser console tab if `/collab` websocket path is transmitting data when you type on the editor.

<img width="1496" alt="изображение" src="https://github.com/user-attachments/assets/3dc43b1c-994d-4335-b609-258abe4945f5">

<img width="1472" alt="изображение" src="https://github.com/user-attachments/assets/c69cc038-5f30-4928-a7f9-afceb09c4de7">

websokets is ok

**Reply by vnetesov** [reply]
After you have enabled the websockets, you need to log in to each page, only then the rights apply. In general, everything works fine, thank you!
